core/src/sumeragi/main_loop.rs

```diff
@@ -1135,8 +1135,28 @@ pub(crate) fn run(
                     .unwrap_or_else(|err| error!("{err}"));
```

Comment:
```suggestion
                    .expect("INTERNAL BUG: Invalid view change proof");
```

this can't fail, can it?